smoke-test/tests/cypress/cypress.config.js

```diff
@@ -0,0 +1,24 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
```

Comment:
We might face a few challenges merging this back. Let's make sure that's accounted for !